plugins/modules/vios_alt_root_vg.py

```diff
@@ -0,0 +1,576 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2018- IBM, Inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: vios_alt_root_vg
+author:
+    - Sreenidhi S(@SreenidhiS1)
+short_description: Create/Cleanup an alternate rootvg disk on a VIOS
+notes:
+    - For executing this module, create a task role with following command
+      mkaccfg -t taskrole -i name=new_task_role,parent=hmcsuperadmin,
```

Comment:
Can you put this command in I() or C() format and also don't specify the task role let the user to define it.